corehq/apps/app_manager/views/modules.py

```diff
@@ -571,8 +571,8 @@ def should_edit(attribute):
 
     handle_media_edits(request, module, should_edit, resp, lang)
     handle_media_edits(request, module.case_list_form, should_edit, resp, lang, prefix='case_list_form_')
-    handle_media_edits(request, module.case_list, should_edit, resp, lang, prefix='case_list-menu_item_')
-
+    if module.case_list:
```

Comment:
This will throw the same error. You want `hasattr(module, 'case_list')`

Reply:
Right. Thank you!